front-end/generate-front-end.js

```diff
@@ -21,26 +21,37 @@ module.exports = (bucket, commitHash, opts) => {
         return collection;
       }
 
-      const storyDetails = url.match(/(.*)\.(.*)\.png/);
+      const storyDetails = url.match(/dextrose-test\/(.*)\.(android|ios|web).?(.*).png/);
       const storyName = storyDetails[1];
       const platform = storyDetails[2];
+      const width = storyDetails[3];
 
       if (!collection[storyName]) {
         names.push(storyName);
         collection[storyName] = {}; // eslint-disable-line no-param-reassign
       }
 
-      collection[storyName][platform] = url; // eslint-disable-line no-param-reassign
+      if (platform === "web") {
+        collection[storyName][platform] = collection[storyName][platform] ? collection[storyName][platform] : {};
+        collection[storyName][platform][width] = url;
+      } else {
+        collection[storyName][platform] = url; // eslint-disable-line no-param-reassign
+      }
 
       return collection;
     }, {});
 
+    const webWidths = Object.entries(shots)[0][1].web;
```

Comment:
might have to be defensive here, need to check for case where web might not exist